app/models/chat_message.rb

```diff
@@ -68,44 +98,6 @@ def rebake!(invalidate_oneboxes: false, priority: nil)
 
     Jobs.enqueue(:process_chat_message, args)
   end
-
-  COOK_FEATURES = {
-    anchor: true,
-    "auto-link": true,
-    bbcode: true,
-    "bbcode-block": true,
-    "bbcode-inline": true,
-    "bold-italics": true,
-    "category-hashtag": true,
-    censored: true,
-    checklist: false,
-    code: true,
-    "custom-typographer-replacements": false,
-    "d-wrap": false,
-    details: false,
-    "discourse-local-dates": true,
-    emoji: true,
-    emojiShortcuts: true,
-    html: false,
-    "html-img": true,
-    "inject-line-number": true,
-    inlineEmoji: true,
-    linkify: true,
-    mentions: true,
-    newline: true,
```

Comment:
Migrated to https://github.com/discourse/discourse-chat/pull/468/files#diff-51e0b19328dd306519dc69e052bdaca6fa7c85233020eca3bb73d0d575026f6dR57